app/fissile.go

```diff
@@ -951,6 +951,14 @@ func (f *Fissile) generateHelmValues(outputDir string, rolesManifest *model.Role
 		_, err = outputFile.Write(buf)
 	}
 	if err == nil {
+		_, err = outputFile.Write([]byte(`kube:
```

Comment:
It is possible that my readability bar is set too low due to years to terrible language abuse:
```golang
	buf, err := yaml.Marshal(map[string]interface{}{
		"env": env,
		"kube": map[string]interface{}{
			"external_ip": "192.168.77.77",
			"storage_class": map[string]interface{}{
				"persistent": "persistent",
				"shared":     "shared",
			},
		},
	})
	if err == nil {
		_, err = outputFile.Write(buf)
	}
```